Src/cmor_grids.c

```diff
@@ -760,11 +762,24 @@ int cmor_set_grid_mapping(int gid, char *name, int nparam,
 
     for (i = 0; i < nattributes - 7; i++) {
         if (cmor_has_grid_attribute(gid, grid_attributes[i]) == 1) {
-            cmor_handle_error_variadic(
-                "Grid mapping attribute %s has not been set, "
-                "you should consider setting it",
-                CMOR_WARNING,
-                grid_attributes[i]);
+            if (strcmp(grid_attributes[i], "standard_parallel") == 0) {
```

Comment:
@mauzey1 would it be useful to note within the code that the settings for the included grid_mappings are consistent with CF 1.12? These will likely to continue to evolve, so we'd want to attempt to keep track of version changes, and note our current baseline

Reply:
Do you want something to the effect of the following in `cmor_grid_valid_mapping_attribute_names`?
```
/* -------------------------------------------------------------------- */
/*   The grid mapping attribute mappings below are consistent with      */
/*   CF 1.12 (https://cfconventions.org/Data/cf-conventions/            */
/*   cf-conventions-1.12/cf-conventions.html#appendix-grid-mappings)    */
/* -------------------------------------------------------------------- */

```

Reply:
Wonderful, thanks